pull_alerts.py

```diff
@@ -109,12 +132,36 @@ def _get_time_window():
     return time_window
 
 
+def actionable(incident):
+    return '#a' in incident.notes
```

Comment:
I would recommend checking to see if `#a` or `#na` is anywhere in each note, instead of an exact match.